lib/assets/javascripts/dmproadmap/forms.js

```diff
@@ -49,7 +49,7 @@ function validatePassword(sPassword) {
 
 // ---------------------------------------------------------------------------
 function validateEmail(sEmail) {
-  var filter = /^[a-zA-Z0-9]+[a-zA-Z0-9_.-]+[a-zA-Z0-9_-]+@[a-zA-Z0-9]+[a-zA-Z0-9.-]+.[a-z]{2,4}$/;
+  var filter = /[^@\s]+@(?:[-a-z0-9]+\.)+[a-z]{2,}$/;
```

Comment:
This JS regex maps the same regex specified at contact_us model https://github.com/JDutil/contact_us/blob/master/app/models/contact_us/contact.rb#L7 

I'm not aware of having in are User models a validator for email...